### DISCUSSION TITLE
Class Loading Issue in 20.0

### DISCUSSION THREAD

**Discussion by schaefa** [graphql-java/graphql-java#3100] [General]
Hi

As others, too, I encountered class loading issue with 20.0. If I recompile the master branch with 'USE_LAMBDA_FACTORY = new AtomicBoolean(false)' this issue is gone.

My question is how am I supposed to set the flag to false w/o that code change?

We are using graphql-java inside an OSGi container (Felix) and I don't see that the method settings this flag (setUseLambdaFactory()) is exposed.

**Comment by bbakerman** [comment]
Just call ` graphql.schema.PropertyDataFetcherHelper#setUseLambdaFactory(false);` - its a publically available static method

**Comment by schaefa** [comment]
I missed that. Thanks - this did the trick.
